### PR TITLE
Expo App package with TS

### PR DESCRIPTION
**Pull request by mauriciord**
Based on https://github.com/entria/entria-fullstack/issues/73

Universal Expo App (iOS|Android|Web) with Typescript.

```
  Expo CLI 3.13.1 environment info:
    System:
      OS: macOS 10.15.2
      Shell: 5.7.1 - /bin/zsh
    Binaries:
      Node: 12.16.1 - /var/folders/cg/776y_8l900qdrgww25ylv47h0000gn/T/yarn--1583372700609-0.6789692417621209/node
      Yarn: 1.21.1 - /var/folders/cg/776y_8l900qdrgww25ylv47h0000gn/T/yarn--1583372700609-0.6789692417621209/yarn
      npm: 6.13.4 - ~/.nvm/versions/node/v12.16.1/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    IDEs:
      Android Studio: 3.1 AI-173.4720617
      Xcode: 11.3.1/11C504 - /usr/bin/xcodebuild
    npmPackages:
      expo: 36.0.2 => 36.0.2
      react: ~16.9.0 => 16.9.0
      react-native: https://github.com/expo/react-native/archive/sdk-36.0.0.tar.gz => 0.61.4
```

### Whats next?
- Another PR with Relay

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,7 +61,9 @@
       "yarn sort-package-json"
     ]
   },
-  "dependencies": {},
+  "dependencies": {
+    "metro-config": "^0.58.0"
```
Comment (jgcmarins) [inline package.json]: maybe it should be inside app's package

**Comment by mauriciord** [comment]
It's draft yet because I need to resolve this React Native render error, so after that, I'm gonna insert typescript, and open another PR to add relay ;)

**Comment by mauriciord** [comment]
Let me resolve the conflicts first :-]

File: package.json [hunk 2]
```diff
@@ -12,22 +12,14 @@
...
-    "app:pod": "yarn workspace @golden-stack/app pod:install",
-    "app:redirect-ports": "yarn workspace @golden-stack/app redirect-ports",
-    "app:relay": "yarn workspace @golden-stack/app relay",
-    "app:shake": "yarn workspace @golden-stack/app shake",
-    "app:start": "yarn workspace @golden-stack/app start",
+    "app:android": "yarn workspace @golden-stack/expo-app expo android",
+    "app:ios": "yarn workspace @golden-stack/expo-app expo ios",
+    "app:relay": "yarn workspace @golden-stack/expo-app relay",
+    "app:start": "yarn workspace @golden-stack/expo-app start",
+    "app:web": "yarn workspace @golden-stack/expo-app expo start -w",
```
Comment (jean-leonco) [inline package.json]: can you rename this to `@golden-stack/app`?

Comment (mauriciord) [reply package.json]: of course

Comment (mauriciord) [reply package.json]: resolved @jean-leonco

File: package.json [hunk 3]
```diff
@@ -69,12 +61,14 @@
...
     "@typescript-eslint/parser": "^2.18.0",
     "concurrently": "^5.0.0",
+    "expo-yarn-workspaces": "^1.2.1",
     "husky": "^4.2.1",
     "jest": "^25.1.0",
     "jest-cli": "25.1.0",
     "jest-junit": "^10.0.0",
     "lerna": "^3.20.2",
     "lint-staged": "^10.0.3",
+    "metro-config": "^0.58.0",
```
Comment (jgcmarins) [inline package.json]: do we need `metro-config` on the root?

Comment (mauriciord) [reply package.json:71]: Yes, because if the metro config is only inside of expo app, the Expo service does not start

**Comment by mauriciord** [comment]
@jean-leonco Could you review ASAP, please? :-]

**Comment by jean-leonco** [comment]
@jgcmarins approved the PR :)

**Comment by jean-leonco** [comment]
@mauriciord Great job :fire:

**Pull request merged**